<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kanji tutor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="pageSelector">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QWidget" name="startPageWidget">
     <widget class="QScrollArea" name="startPageScrollArea">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>0</y>
        <width>391</width>
        <height>481</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="startPageScrollWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>374</width>
         <height>481</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="categoryPageWidget">
     <widget class="QPushButton" name="categoryPageBackButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QLabel" name="categoryPageNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название категории</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="categoryPageGridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="categoryPageKanjiButton">
         <property name="text">
          <string>Список всех иероглифов</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="categoryPageTestsButton">
         <property name="text">
          <string>Тесты для проверки</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="categoryPageLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>381</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="categoryPageLessonsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>381</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lessonPageWidget">
     <widget class="QPushButton" name="lessonPageBackButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QLabel" name="lessonPageNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название категории</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="lessonPageTabWidget">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>40</y>
        <width>411</width>
        <height>441</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="lessonPageKanjiTab">
       <attribute name="title">
        <string>Список</string>
       </attribute>
       <widget class="QListWidget" name="lessonPageKanjiList">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>7</y>
          <width>382</width>
          <height>401</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="lessonPageTestsTab">
       <attribute name="title">
        <string>Упражнения</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="lessonPageRevisionTab">
       <attribute name="title">
        <string>Повторение</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="categoryKanjiPage">
     <widget class="QPushButton" name="categoryKanjiPageBackButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QLabel" name="categoryKanjiPageNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название категории</string>
      </property>
     </widget>
     <widget class="QListWidget" name="categoryKanjiPageKanjiList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>382</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="menuItemExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="menuItemAbout"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Профиль</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="menuItemExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="menuItemAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
